woocommerce/woocommerce-android
# Issue/13136 manage stores visibility

Closes: #13136 


### Description

Adds a new screen to select or unselect which stores user wants to access through site picker. 

The current PR is only UI changes. Selecting or unselecting sites won't have any impact on the displayed site picker list. 

### Testing information

1. Log into the app using WP.com credentials with an account that has multiple stores
2. Go to the site picker
3. Tap `Edit Store` button
4. Check all Woo sites are displayed and checked
5. Uncheck one site and see how the save button is enabled
6. Check again the same site and see how save is disabled

### The tests that have been performed

The above

### Images/gif


https://github.com/user-attachments/assets/c64f79be-e1f7-4151-9bdc-38aeb1e1e44d


## Reviewer (or Author, in the case of optional code reviews):

Please make sure these conditions are met before approving the PR, or request changes if the PR needs improvement:
